src/status_im/contexts/shell/qr_reader/view.cljs

```diff
@@ -88,7 +88,7 @@
     (load-and-show-profile scanned-text)
 
     (eth-address? scanned-text)
-    (debounce/debounce-and-dispatch [:navigate-to :wallet-accounts scanned-text] 300)
+    (debounce/debounce-and-dispatch [:navigate-to :wallet.accounts scanned-text] 300)
```

Comment:
should it not be `:screen/wallet.accounts` now?

Reply:
Done! thanks for spotting it